docs/source/examples/Detecting Events.mystnb

```diff
@@ -0,0 +1,386 @@
+---
+jupytext:
+  text_representation:
+    extension: .mystnb
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Detecting Events
+
+It is a well-known fact that launching a satellite is a captial-intensive and fuel-exhaustive process.
+Moreover, maintaining high accuracy and precision in any satellite orbit analysis is paramount
+to be able to comprehend helpful information from them.
+
+Detecing some peculiar phenomena associated with satellites, which we call "events", could provide
+beneficial insights about their orbit dynamics for further treatment. While some could provide
+critical scientific information and help us formulate efficient space strategies and policies,
+the potentially disastrous ones, like satellite collisions, could help us take further steps to prevent
+such contingencies.
+
+This notebook gives a glimpse of `poliastro`'s event detection capabilities. The procedure to track
+an event during an orbit's propagation is fairly simple:
+
+1. Instantiate the desired event class/classes.
+2. Pass the `Event` object(s) as an argument to the `cowell` propagation method.
+3. Detect events! Optionally, the `terminal` and `direction` attributes can be set as required.
+
+```{code-cell} ipython3
+# Imports
+import numpy as np
+from numpy.linalg import norm
+import matplotlib.pyplot as plt
+
+import astropy
+import astropy.units as u
+from astropy.time import Time
+from astropy.coordinates import CartesianRepresentation, get_body_barycentric_posvel
+
+from poliastro.bodies import Earth, Sun
+from poliastro.constants import H0_earth, rho0_earth
+from poliastro.earth import EarthSatellite
+from poliastro.earth.plotting import GroundtrackPlotter
+from poliastro.frames import Planes
+from poliastro.plotting import StaticOrbitPlotter
+
+from poliastro.core.events import eclipse_function
```

Comment:
Similarly to other notebooks, we could import the utilities just at the very first cells of the sections where they are used. This is useful for the user to learn in which sub-package or module is the feature located and how to apply it.

Reply:
Thanks! Now, the common imports are at the top and specific imports are in specific cells.